src/components/manage/Contents/Contents.jsx

```diff
@@ -454,7 +454,7 @@ class Contents extends Component {
    */
   onSelectAll() {
     this.setState({
-      selected: map(this.state.items, (item) => item['@id']),
+      selected: map(this.state.items, item => item['@id']),
```

Comment:
I am wondering why this is changed back and forth I guess by prettier. Any idea?

Reply:
Outdated, but yeah it was prettier-related